OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -448,12 +450,7 @@ public bool CanEnterTargetNow(Actor self, Target target)
 			if (target.Positions.Any(p => self.World.ActorMap.GetActorsAt(self.World.Map.CellContaining(p)).Any(a => a != self && a != target.Actor)))
 				return false;
 
-			var res = target.Actor.TraitOrDefault<Reservable>();
-			if (res == null)
-				return true;
-
-			UnReserve();
-			Reservation = res.Reserve(target.Actor, self, this);
+			MakeReservation(target.Actor);
```

Comment:
*PLEASE CHECK THIS*

The old code would bail early if the target was unreservable. This means any old reservation it had would not be cancelled, even though presumably it was now heading to a new target. This seems highly suspicious and so I have changed it.

Frankly - a method named `CanEnterTargetNow` is something I expect to be side-effect free - whereas it will actually make a reservation of the target you are checking. :/